ocs_ci/ocs/node.py

```diff
@@ -1016,6 +1110,19 @@ def node_replacement_verification_steps_user_side(old_node_name, new_node_name):
         log.warning("Not all the pods in running state")
         return False
 
+    new_osd_pod = get_node_pods(new_node_name, pods_to_search=pod.get_osd_pods())[0]
+    if not new_osd_pod:
+        log.warning("Didn't find any osd pods running on the new node")
+        return False
```

Comment:
do you think we should throw exceptions in this function for failures instead of returning boolean and remove the assert in the test case?